modular_skyrat/modules/cryosleep/code/cryopod.dm

```diff
@@ -329,8 +332,40 @@ GLOBAL_LIST_EMPTY(cryopod_computers)
 		to_chat(user, span_notice("Dead people can not be put into cryo."))
 		return
 
+// Allows admins to enable players to override SSD Time check.
+	if(allow_player_override)
+		if(tgui_alert(user, "Would you like to place [target] into [src]?", "Place into Cryopod?", list("Yes", "No")) != "No")
+			to_chat(user, span_danger("You put [target] into [src]. [target.p_theyre(capitalized = TRUE)] in the cryopod."))
+			log_admin("[key_name(user)] has put [key_name(target)] into a overridden stasis pod.")
+			message_admins("[key_name(user)] has put [key_name(target)] into a overridden stasis pod. [ADMIN_JMP(src)]")
+
+			add_fingerprint(target)
+
+			close_machine(target)
+			name = "[name] ([target.name])"
```

Comment:
should return at the end here, no?

Reply:
No only because it handles the remaining work the script would continue. Allowing us to bypass all the extra checks as the player enters. close_machine() is what inserts the player then starts the cryo timer.